include/multipass/cli/command.h

```diff
@@ -81,6 +86,24 @@ class Command
         {
             return on_success(reply);
         }
+        else
+        {
+            const auto tokens = multipass::utils::split(context.peer(), ":");
+            if (tokens[0] == "unix")
```

Comment:
Could we do the same for network sockets?

Reply:
Sure, it's possible, but I think that may be beyond the scope of this PR.  But if you want to wrap that check into this PR so we don't lose the task to do this, I'm fine with that too:)